main.py

```diff
@@ -0,0 +1,77 @@
+from fastapi import FastAPI, Depends, HTTPException
+from sqlalchemy.orm import Session
+
+import schemas
+from crud import (
+    get_all_authors,
+    create_author,
+    get_author_detail,
+    get_all_books,
+    create_book,
+)
+from database import SessionLocal
+
+
+app = FastAPI()
+
+API_PREFIX = "/api/v1/"
+
+
+def get_db() -> Session:
+    db = SessionLocal()
+
+    try:
+        yield db
+    finally:
+        db.close()
+
+
+@app.get(API_PREFIX + "authors/", response_model=list[schemas.Author])
+def read_all_authors(
+        skip: int | None = None,
+        limit: int | None = None,
+        db: Session = Depends(get_db)
+) -> list[schemas.Author]:
+    return get_all_authors(db=db, skip=skip, limit=limit)
+
+
+@app.post(API_PREFIX + "authors/", response_model=schemas.Author)
+def create_author(
+        author: schemas.AuthorCreate,
+        db: Session = Depends(get_db)
+) -> schemas.Author:
+    return create_author(db=db, book=author)
```

Comment:
In the create_author function, you're passing a parameter named 'book' to the create_author function from the crud module, but it should be 'author'. It seems like a small typo. The code should be `return create_author(db=db, author=author)` instead of `return create_author(db=db, book=author)`